lms/djangoapps/courseware/access_response.py

```diff
@@ -144,6 +144,31 @@ def __init__(self, start_date, display_error_to_user=True):
         )
 
 
+class StartDateEnterpriseLearnerError(AccessError):
+    """
+    Access denied because the course has not started yet and the user
+    is not staff.  This user is an enterprise learner and enrolled in the requested course.
+    """
+    def __init__(self, start_date, display_error_to_user=True):
+        """
+        Arguments:
+            display_error_to_user: If True, display this error to users in the UI.
+        """
+        error_code = "course_not_started_enterprise_learner"
+        if start_date == DEFAULT_START_DATE:
+            developer_message = "Course has not started"
```

Comment:
Maybe something in here about enterprise?

Reply:
👍 agreed.  FWIW this will be more useful as a developer message than a user message, since the user experience is not one that suggests an error occurred.